plugins.json

```diff
@@ -299,6 +299,18 @@
         "remote_install": true,
         "compatible_version": ">=1.0.37"
     },
+    "themeplus-gray": {
+        "title": "KB Themeplus Gray",
+        "version": "0.0.1",
+        "author": "Décio Benicio",
+        "license": "MIT",
+        "description": "Modern and responsive theme for kanboard.",
+        "homepage": "https://github.com/deciobenicio/kb-themeplus-stgray",
+        "readme": "https://raw.githubusercontent.com/deciobenicio/kb-themeplus-stgray/master/README.md",
+        "download": "https://github.com/deciobenicio/kb-themeplus-stgray/archive/master.zip",
```

Comment:
The archive should have the directory structure mentioned here: https://github.com/kanboard/website#how-to-add-a-new-plugin-to-the-list

Reply:
@fguillot , I made a fork and a PR into Decio's repository, trying to follow your orientation. Please, take a look in his repo and see if now is ok.